app/views/sections/items/ItemWineOperationsChoiceView.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import models.response.referenceData.WineOperations
+@import viewmodels.LegendSize
+@import viewmodels.helpers.ItemWineOperationsChoiceHelper.formatCheckboxesForDisplay
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukCheckboxes: GovukCheckboxes,
+        govukButton: GovukButton,
+        h1: components.h1,
+        h2: components.h2,
+        continueOrExit: components.continueOrExit
+)
+
+
+@(form: Form[_], action: Call, values: Seq[WineOperations])(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("itemWineOperationsChoice.title")), maybeShowActiveTrader = maybeShowActiveTrader(request)) {
+
+    @formHelper(action = action, 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
```

Comment:
error message does not show above the checkboxes, it only shows at top of page.
<img width="1302" alt="image" src="https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/15212062/bb42fe47-060f-4d6a-98b6-21575b63557f">

Reply:
Updated https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/199/commits/128bbca894d5c7506a7996c15fe00b91da42f247